RELEASES.md

```diff
@@ -2,8 +2,31 @@
 
 ## Pending Release
 
-- Removed static API handler: `subnetevm.decodeGenesis`, `subnetevm.buildGenesis`
-- Demoted unnecessary error log in `core/txpool/legacypool.go` to warning, displaying unexpected but valid behavior.
```

Comment:
this will be in v0.7.6

Reply:
Are you sure? Those are listed in the release notes for 0.7.5 https://github.com/ava-labs/subnet-evm/releases/tag/v0.7.5